Refresh.GameServer/Types/Levels/Categories/CategoryService.cs

```diff
@@ -48,7 +48,7 @@ public class CategoryService : EndpointService
             IconHash = "g820607",
             FontAwesomeIcon = "heart",
         },
-        new LevelCategory("mostLiked", "thumbs", false, nameof(GameDatabaseContext.GetHighestRatedLevels))
+        new LevelCategory("mostLiked", new[] { "thumbs", "highestRated" }, false, nameof(GameDatabaseContext.GetHighestRatedLevels))
```

Comment:
Aliases would probably fit better down below in the initializer, not as a property of the constructor.

Reply:
I did it this way to keep the code simpler at the slots endpoint, as instead of checking for "route or one any of the aliases" it checks for "any of the routes"

Reply:
Hmm, that definitely makes it more annoying to read the constructor but if that's the purpose then I guess it's okay.